jest.config.js

```diff
@@ -1,22 +1,23 @@
 const { compilerOptions } = require('./tsconfig.json');
 
+const tsJestOptions = {
+	isolatedModules: true,
+	tsconfig: {
+		...compilerOptions,
+		declaration: false,
+		sourceMap: false,
+		skipLibCheck: true,
+	},
+};
+
 /** @type {import('jest').Config} */
 module.exports = {
 	verbose: true,
-	preset: 'ts-jest',
 	testEnvironment: 'node',
 	testRegex: '\\.(test|spec)\\.(js|ts)$',
 	testPathIgnorePatterns: ['/dist/', '/node_modules/'],
-	globals: {
-		'ts-jest': {
-			isolatedModules: true,
-			tsconfig: {
-				...compilerOptions,
-				declaration: false,
-				sourceMap: false,
-				skipLibCheck: true,
-			},
-		},
+	transform: {
+		'^.+\\.ts$': ['ts-jest', tsJestOptions],
```

Comment:
Shouldn't this match `testRegex`?

Reply:
`testRegex` is used to determine which files contain tests. This is used to tell jest for what files, the code needs to be compiled before it can be used. `.js` files don't need this transform (at least in our repo).